python/ray/train/tests/test_mosaic_trainer.py

```diff
@@ -0,0 +1,208 @@
+import pytest
+
+import torch
+import torch.utils.data
+
+import torchvision
+from torchvision import transforms, datasets
+
+from torchmetrics.classification.accuracy import Accuracy
+from composer.core.evaluator import Evaluator
+from composer.models.tasks import ComposerClassifier
+import composer.optim
+from composer.algorithms import LabelSmoothing
+from composer.loggers.logger_destination import LoggerDestination
+from composer.core.state import State
+from composer.loggers import Logger
+from composer.core.callback import Callback
+
+
+import ray
+from ray.air.config import ScalingConfig
+import ray.train as train
+from ray.air import session
+from ray.train.mosaic import MosaicTrainer
+
+scaling_config = ScalingConfig(num_workers=2, use_gpu=False)
+
+
+def trainer_init_per_worker(config):
```

Comment:
We can just replace this entire thing with `from ray.train.examples.mosaic_cifar10_examples import trainer_init_per_worker`.

That way we don't have to duplicate the code